alacritty/src/display/mod.rs

```diff
@@ -969,7 +969,9 @@ impl Display {
         // Frame event should be requested before swaping buffers, since it requires surface
         // `commit`, which is done by swap buffers under the hood.
         #[cfg(all(feature = "wayland", not(any(target_os = "macos", windows))))]
```

Comment:
I think this `cfg` can be removed? Sure it's a runtime check more if Wayland is disabled but it should basically be free and the function exists everywhere now.

Otherwise I'd suggest adding cfgs to all the `!self.is_wayland` places too, we should just be consistent.